From c5cdefa21e0ebd1e86aace4e07ea1114b840ff00 Mon Sep 17 00:00:00 2001
From: Steven Rostedt <srostedt@redhat.com>
Date: Fri, 22 Jun 2012 07:25:14 -0400
Subject: [254/256] spinlock/rt: Include bug.h in spinlock_rt.h to satisfy
 BUG() usage

Some archs (powerpc) failed to compile because of the reference to
BUG() in spinlock_rt.h. Although bug.h is included in many headers,
if spinlock.h is included in a C file before that, it will fail to
compile.

Signed-off-by: Steven Rostedt <rostedt@goodmis.org>
---
 include/linux/spinlock_rt.h |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/include/linux/spinlock_rt.h b/include/linux/spinlock_rt.h
index 3b555b4..0618387 100644
--- a/include/linux/spinlock_rt.h
+++ b/include/linux/spinlock_rt.h
@@ -5,6 +5,8 @@
 #error Do not include directly. Use spinlock.h
 #endif
 
+#include <linux/bug.h>
+
 extern void
 __rt_spin_lock_init(spinlock_t *lock, char *name, struct lock_class_key *key);
 
